joining.html

```diff
@@ -0,0 +1,217 @@
+        <li>Do you have <b><u>under</u></b> 15M SP or a vouch from a
+            member?
+            (<a href="#why-sp-max">why?</a>)
+        </li>
+        <li>Do you have <b>over</b> 1M SP, or if not, have you run the
+            career missions?
+            (<a href="#why-sp-min">why?</a>)
+        </li>
+        <li>Do you have TeamSpeak installed?</li>
+      </ul>
+    </p>
+
+    <h2>
+      <a name="applying" href="#applying">#</a>
+      Applying
+    </h2>
+    <p>
+      Visit us in our in-game public channel: <b>[Of Sound Mind]</b> (yep, with
+      the brackets included) and ask to talk to a recruiter. Please be patient,
+      especially if you're not USTZ - you may need to wait a while for a
+      recruiter to be available. Your recruiter will collect some basic info
+      and start an applicant file for you, then help you through the next steps.
+    </p>
+
+    <h2>
+      <a name="trial-fleet" href="#trial-fleet">#</a>
+      Trial Fleet
+      <sup>(<a href="#why-trial-fleet">why?</a>)</sup>
+    </h2>
+    <p>
+      Your recruiter will help you find a scheduled trial fleet. We run these
+      every week or so, but can also sometimes run them on demand if folks are
+      available. Trial fleets will pick you up in
+      <a href="https://evemaps.dotlan.net/map/Placid/Orvolle">Orvolle</a> in
+      <b>Orvolle I - Federation Navy Assembly Plant</b>. You will need to have
+      Teamspeak installed. The fleet commander will give you access to our
+      Teamspeak for the duration of the fleet.
+    </p>
+    <p>
+      You do not need to bring a ship, but if you want to, you are welcome to
+      bring anything cruiser-sized or smaller. If you do not have or want to
+      bring a ship, free ships appropriate for all SP levels are available.
+    </p>
+    <p>
+      The trial fleet generally lasts about <b>90 minutes</b>, and will drop you
+      off in Orvolle when it finishes.
+    </p>
+
+    <h2>
+      <a name="esi-knife" href="#esi-knife">#</a>
+      ESI Knife Audit
+    </h2>
+    <p>
+      Your recruiter will ask you to sign up for
+      <a href="https://esiknife.com/">ESI Knife</a>, which is a website that
+      allows your recruiter to look at your character's history, assets, and
+      so on. This is a standard security precaution for Eve corps and does
+      <b><u>not</u></b> grant any access to do anything other than view
+      details.
+    </p>
+
+    <h2>
+      <a name="interview" href="#interview">#</a>
+      Interview
+    </h2>
+    <p>
+      After your trial fleet, your recruiter will work with you to schedule
+      an interview over Teamspeak. This interview generally lasts half an
+      hour at most. You'll probably be asked about at least these things:
+      <ul>
+        <li>Your experiences in Eve</li>
+        <li>Your corp history</li>
+        <li>Your trial fleet experience</li>
+      </ul>
+    </p>
```

Comment:
What about the written interview?

Reply:
Changed to clarify that it can be written, verbal, or both simultaneously, since I know some recruiters prefer one type or the other.